Ginger/GingerCoreNET/RecordingLib/RecordingManager.cs

```diff
@@ -58,6 +66,7 @@ public RecordingManager(List<ApplicationPOMModel> lstApplicationPOM, BusinessFlo
                 }
 
                 BusinessFlow = bFlow;
+                PlatformDriver.CurrentBusinessFlow = bFlow;
```

Comment:
you already passing Context, why also CurrentBusinessFlow?

Reply:
while initial implementation i was trying to check if the businessflow is changed or not so i added this property, which might be used in future